openmetadata-ui/src/main/resources/ui/src/utils/TableProfilerUtils.ts

```diff
@@ -76,7 +81,7 @@ export const calculateSystemMetrics = (
     return {
       ...item,
       stackId: stackId,
-      latestValue: operation?.rowsAffected ?? 0,
+      latestValue: operation?.rowsAffected,
```

Comment:
What if the latest value is undefined? IMO we should have the default value?

Reply:
That's the main change we wanted as per ticket, if api doesn't have data, we will not show graph/point in graph.